aibolit/patterns/non_final_attribute/non_final_attribute.py

```diff
@@ -0,0 +1,26 @@
+import javalang
+
+
+class NonFinalAttribute:
+
+    def __init__(self):
+        pass
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
```

Comment:
@paulodamaso This one is copy-pasted over the project, so I created a ticket for that https://github.com/yegor256/aibolit/issues/131